### PR TITLE
VTE Measure data requirements testscript

### PR DESCRIPTION
**Pull request by mgramigna**
- Added VTE Measure and VTE Libraries including related artifacts to reporting resources
- Renamed colon cancer measure data requirements script to
"col-data-requirements"
- Added testscript for VTE measure data requirements

`bundle exec rake crucible:execute[<base stu3 url>,stu3,vte-data-requirements]`

### CODE REVIEW

File: lib/tests/testscripts/scripts/reporting/vte-data-requirements.xml [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    <publisher value="MITRE"/>
+    <contact>
+        <name value="Matthew Gramigna"/>
+        <telecom>
+            <system value="email"/>
+            <value value="mgramigna@mitre.org"/>
+            <use value="work"/>
+        </telecom>
+    </contact>
+    <description value="Tests using XML format to execute the $data-requirements operation operation. The destination server must support the $data-requirements operation on the Measure resource."/>
```
Comment (okeefm) [inline lib/tests/testscripts/scripts/reporting/vte-data-requirements.xml]: Technically this TestScript tests the `JSON` formatted requests/responses for `$data-requirements`

Comment (mgramigna) [reply lib/tests/testscripts/scripts/reporting/vte-data-requirements.xml]: Whoops. Just pushed a small update to description wording.

**Review by okeefm — APPROVED** [review]
I didn't review all of the artifacts that get uploaded, just the testscript itself, but the test passes when run against my local cqf-ruler instance. Looks good to me.

**Review by tstrass — APPROVED** [review]
LGTM

**Pull request merged**